.github/workflows/macos-xpack-filebeat.yml

```
@@ -13,6 +13,9 @@ on:
 env:
   BEAT_MODULE: 'x-pack/filebeat'
 
+permissions:
+  contents: read
```

```suggestion
  contents: read
  actions: write
```

Do we need this because of `uses: v1v/otel-upload-test-artifact-action@v2`?